src/popup/utils/constants.ts

```diff
@@ -594,3 +596,22 @@ export const ALLOWED_ICON_STATUSES = [
   'success',
   'warning',
 ] as const;
+
+export const WEB_SOCKET_CHANNELS = {
+  Transactions: 'Transactions',
+  MicroBlocks: 'MicroBlocks',
+  KeyBlocks: 'KeyBlocks',
+  Object: 'Object',
+};
+
+export const WEB_SOCKET_SUBSCRIBE = 'Subscribe';
+export const WEB_SOCKET_UN_SUBSCRIBE = 'Unsubscribe';
+
+export const WEB_SOCKET_CHANNEL_NAMES = [
```

Comment:
This can be removed as it's only used within the type definitions. See my further comment I did there on how you can actually create the type for the channel name.